### ISSUE TITLE
When executing the Segment, after the box selects the polygon, I want to delete the point cloud by a certain depth instead of deleting all the boxes selected.

### ISSUE THREAD

**Issue by jianhuaz** [CloudCompare/CloudCompare#2071]
When executing the Segment, after the box selects the polygon, I want to delete the point cloud by a certain depth instead of deleting all the boxes selected. 
After clicking segmennt out, enter a value to delete only the point cloud data within this value range. 
The main reason is that the point cloud map generated by the multi-line LiDAR and the point cloud data generated at the top are also generated, and when drawing the lane, the top point cloud is often used as the lane line point, resulting in lane errors. Therefore, it is necessary to manually delete the top point cloud data.

**Comment by dgirardeau** [comment]
Meanwhile, could you use the 'Cross section' tool to isolate the layer with the right depth? If it's a matter of keeping the whole cloud for context, you can also keep the original cloud visible but only select the segmented layer when using the scissors tool.

But I understand that having a simple 'depth' control in the scissors tool would be more efficient.

**Comment by jianhuaz** [comment]
Indeed, the clipped point cloud Map is only used by Vector Map Builder, and the original point cloud map is provided to Autoware.
